website/content/api/stocks/_index.md

```diff
@@ -1,6 +1,67 @@
 ---
-title: stocks
-keywords: ""
-excerpt: ""
+title: Introduction to the Stocks SDK
+keywords: "stocks, stock, options, option, call, put, earnings, calendar, how-to, guide, scripts, fundamental, analysis, technical, behavioural, analyst, equity, research, api, sdk, application, python, notebook, jupyter"
+excerpt: "This guide introduces the Stocks menu in the context of the OpenBB SDK."
 geekdocCollapseSection: true
 ---
+
+The <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/stocks/" target="_blank">Stocks menu</a> wraps all of the functions from the OpenBB Terminal application into a powerful SDK, enabling users to work with the data in a flexible environment that can be fully customized to meet the needs of any user. Code completion and contextual help makes the SDK is easy to use. Navigation is very similar to operating the CLI Terminal Application. 
+
+## How to use the Stocks SDK
+
+Start a Jupyter Notebook with as little as one line of code:
+
+`from openbb_terminal.api import openbb`</br>
+
+In the next cell, the code complete will activate after the `.` is entered, showing the submenus and functions at the root level of the <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/stocks/" target="_blank">Stocks menu</a>.
+
+![Stocks Menu](https://user-images.githubusercontent.com/85772166/195198260-fe1c4c65-2dc2-4001-94ff-f09e0b0ff431.png)
+
+The contents of the menu is printed by running a cell with: `openbb.stocks`
+
+![Stocks Menu](https://user-images.githubusercontent.com/85772166/195198410-0b44ee92-2c79-4efc-8490-a2de3173cfec.png)
```

Comment:
There is too much to print here, you only show a fraction of what we have with this. I'd suggest mentioning each menu we have similarly to what we already do here https://openbb-finance.github.io/OpenBBTerminal/terminal/stocks/. Actually, you should copy + paste that and change the links.